libs/ngx-charts-on-fhir/src/lib/data-layer-list/dataset-options/dataset-options.component.html

```diff
@@ -29,10 +29,24 @@
         ><input matSliderThumb formControlName="pointRadius"
       /></mat-slider>
     </div>
-    <section *ngIf="this.form.value.type === 'line'">
-      <h3>Line Chart Options</h3>
-      <mat-slide-toggle formControlName="interpolation">Cubic interpolation</mat-slide-toggle>
-      <mat-slide-toggle formControlName="fill">Fill area under line</mat-slide-toggle>
-    </section>
+    <div>
+      <section *ngIf="this.form.value.type === 'line'">
+        <h3>Line Chart Options</h3>
+        <mat-slide-toggle formControlName="interpolation">Line Smoothing</mat-slide-toggle>
+        <mat-slide-toggle formControlName="fill">Fill area under line</mat-slide-toggle>
+      </section>
+    </div>
   </ng-container>
 </form>
+<ng-template #simpleOptions>
+  <form [formGroup]="form">
+    <ng-container *ngIf="this._dataset?.yAxisID != 'medications'">
+      <div>
+        <section *ngIf="this.form.value.type === 'line'">
+          <h3>Line Chart Options</h3>
+          <mat-slide-toggle formControlName="interpolation">Line Smoothing</mat-slide-toggle>
+        </section>
+      </div>
+    </ng-container>
+  </form>
+</ng-template>
```

Comment:
I don't think we need the extra div, section, or "Line Chart Options" heading here because there's only one option. Let's simplify it.

Reply:
Okay I will simplify it.